config/manager/kustomization.yaml

```diff
@@ -13,3 +13,4 @@ configMapGenerator:
 images:
 - name: controller
   newName: public.ecr.aws/m7r9p7b3/aws-gateway-controller
+  newTag: v0.0.12
```

Comment:
can you remove this line? I think it is a side effect of you running `make build-deploy` for test

Reply:
Done next commit!